src/regions/ca/info/ca-ns/clinics.fr.md

```diff
@@ -16,4 +16,6 @@ Si vous avez répondu oui à deux questions ou plus, appelez le service 811 pour
 
 Si vous éprouvez d’autres symptômes qui vous inquiètent, appelez le 811 pour parler à une infirmière.
 
+Les symptômes d’une fièvre sont les suivants: frissons, sueurs, douleurs musculaires, étourdissements.
```

Comment:
+ the temperature maybe?

Reply:
Normally, the temperature is already mentioned above. This information is a copy/paste from the official NS website.